src/components/modules/puzzles/puzzlesBanner/squares/ContentSquares.tsx

```diff
@@ -0,0 +1,232 @@
+
+    ::before {
+        content: ' ';
+        display: block;
+        width: 6px;
+        background-color: ${neutral[20]};
+        border-top: ${border};
+        height: 100%;
+        transform: translateY(-2px) skewY(-30deg);
+    }
+`;
+
+const contentSquare = css`
+    z-index: 2;
+    ${headline.xxxsmall({ fontWeight: 'bold' })};
+    color: ${neutral[7]};
+    border-bottom: ${border};
+    padding-top: 0;
+    box-shadow: ${boxShadow};
+    min-width: ${space[24]}px;
+    min-height: ${space[24]}px;
+
+    p {
+        padding: ${space[1]}px;
+        padding-left: 0;
+        margin: 0;
+        margin-left: -2px;
+    }
+
+    ${from.tablet} {
+        width: 154px;
+        height: 154px;
+    }
+
+    ${from.desktop} {
+        ${headline.xxsmall({ fontWeight: 'bold' })};
+        width: 180px;
+        height: 180px;
+    }
+`;
+
+const bottomLeftOnMobile = css`
+    ${until.tablet} {
+        grid-row: 2;
+        grid-column: 1;
+        width: 120px;
+        height: 120px;
+    }
+`;
+
+const bottomRightOnMobile = css`
+    ${until.tablet} {
+        grid-row: 2;
+        grid-column: 3;
+        width: 152px;
+        height: 152px;
+    }
+`;
+
+const topRightOnMobile = css`
+    ${until.tablet} {
+        justify-self: center;
+        width: 96px;
+        height: 96px;
+    }
+`;
+
+const downShiftedSquare = css`
+    grid-column: 3;
+    grid-row: 1;
+
+    ${from.tablet} {
+        grid-column: 2;
+        transform: translateY(86px);
+    }
+
+    ${from.desktop} {
+        transform: translateY(80px);
+    }
+`;
+
+const qrCodeSquare = css`
+    border-bottom: none;
+    box-shadow: none;
+
+    & * {
+        box-shadow: none;
+    }
+
+    ${until.tablet} {
+        display: none;
+    }
+`;
+
+const qrCodeContainer = css`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-bottom: ${space[3]}px;
+
+    ${until.desktop} {
+        img {
+            width: 84px;
+            height: 84px;
+        }
+    }
+`;
+
+const textHighlight = css`
+    background-color: ${brandAlt[400]};
+`;
+
+const contentSquaresGrid = css`
+    position: absolute;
+    bottom: 0;
+    display: grid;
+    left: ${space[1]}px;
+
+    ${until.tablet} {
+        bottom: ${space[4]}px;
+        grid-template-columns: 1fr minmax(1px, 64px) 1fr;
+        grid-row-gap: 20px;
+        margin: 0 ${space[4]}px;
+    }
+
+    ${from.mobileLandscape} {
+        left: ${space[3]}px;
+    }
+
+    ${from.tablet} {
+        left: unset;
+        right: 0;
+        grid-template-columns: repeat(3, 150px);
+        grid-row-gap: 76px;
+        grid-column-gap: 2px;
+    }
+
+    ${from.desktop} {
+        grid-template-columns: ${withIECompatibleGap(
+            ['180px', 'minmax(0, 76px)', '180px', '180px'],
+            `${space[6]}px`,
+        )};
+        grid-template-rows: ${withIECompatibleGap(['1fr', '1fr'], `${space[24]}px`)};
+        grid-row-gap: 0;
+        grid-column-gap: 0;
+    }
+`;
+
```

Comment:
Given there's quite a bit of styling in this file, it might help clarity to move it into a separate file. Your call though!

Reply:
I think I might do this in the next PR; I'm extracting some of it to make the 'square side' into a separate component, but it's still quite a bit.